counsel.el

```diff
@@ -2954,10 +2985,16 @@ Return pair of corresponding strings (SWITCHES . SEARCH-TERM)."
   "Construct a complete `counsel-ag-command' as a string.
 EXTRA-ARGS is a string of the additional arguments.
 NEEDLE is the search string."
-  (format counsel-ag-command
-          (if (string-match " \\(--\\) " extra-args)
-              (replace-match needle t t extra-args 1)
-            (concat extra-args " " needle))))
+  (counsel--format counsel-ag-command
+          (if (listp counsel-ag-command)
+              (if (string-match " \\(--\\) " extra-args)
+                  (thread-first (replace-match "%s" t t extra-args 1)
```

Comment:
Please try to avoid the `thread-first` macro, which was added in Emacs 25.1.

Reply:
Thanks